numpy/_core/src/umath/string_buffer.h

```diff
@@ -146,45 +275,37 @@ struct Buffer {
         case ENCODING::UTF32:
             CHECK_OVERFLOW(index * sizeof(npy_ucs4));
             return getchar<enc>((unsigned char *) (buf + index * sizeof(npy_ucs4)), &bytes);
+        case ENCODING::UTF8:
+            // NOT IMPLEMENTED
+            assert(0);
```

Comment:
Should we maybe delete `operator[]` altogether if it's not okay to use with UTF8?

Reply:
Good call, deleting it seems not to break anything.